Khan/wonder-blocks
# WB-314: Fix OneColumnModal and TwoColumnModal on mobile

Fixes [WB-314](https://khanacademy.atlassian.net/browse/WB-314).  We needed to set `height: 100%` on the modal contents so that it occupies all of the space on mobile.

Test Plan:
- load http://localhost:6060/#!/Launcher
- resize the browser width to be narrow to force mobile rendering
- open the OneColumnModal, see that the footer is at the bottom
- open the TwoColumnModal, see that the footer is at the bottom

OneColumnModal Before:
![screen shot 2018-07-16 at 8 25 36 pm](https://user-images.githubusercontent.com/1044413/43018052-5304cc26-8c26-11e8-95ab-eb28d2438caf.png)

OneColumnModal After (disregard the differences in the internal layout):
<img width="777" alt="screen shot 2018-07-20 at 2 10 19 pm" src="https://user-images.githubusercontent.com/1044413/43018147-a8cc0bb0-8c26-11e8-8329-accee00f9d6f.png">

TwoColumnModal Before:
![screen shot 2018-07-16 at 8 24 50 pm](https://user-images.githubusercontent.com/1044413/43018091-7cb444c0-8c26-11e8-92c4-ba0a160416d3.png)

TwoColumnModal After:
<img width="777" alt="screen shot 2018-07-20 at 2 10 15 pm" src="https://user-images.githubusercontent.com/1044413/43018154-acd14838-8c26-11e8-94e7-eabd8cd65071.png">
